src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -87,14 +96,18 @@ internal object AutoMend : Module(
             }
             paused = false
 
+            if (!shouldMend() && autoDisableComplete) {
+                MessageSendHelper.sendChatMessage("$chatName Mending completed! disabling")
+                disable()
+            }
```

Comment:
No thank you, don't put the chat message here. Modules have a ChatToggle option for that reason

Reply:
then should i remove these lines ?
```java
if (autoDisableExp) {
    MessageSendHelper.sendWarningMessage("$chatName No XP in hotbar, disabling")
    disable()
}
                        ```

Reply:
> then should i remove these lines ?

Remove the send chat message